app/src/main/java/org/systers/mentorship/view/fragments/RequestPagerFragment.kt

```diff
@@ -45,7 +45,31 @@ class RequestPagerFragment: BaseFragment() {
             requestsList = it.getParcelableArrayList(Constants.REQUEST_LIST)
             emptyListText = it.getString(Constants.REQUEST_EMPTY_LIST_TEXT)
         }
+        setUI()
+    }
+
+    private val openRequestDetail: (Relationship) -> Unit =
+            { requestDetail ->
+                val intent = Intent(activity, RequestDetailActivity::class.java)
+                intent.putExtra(Constants.RELATIONSHIP_EXTRA, requestDetail)
+                startActivityForResult(intent, 1000)
+            }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        val id = data!!.getStringExtra("requestid")
```

Comment:
Here also, Constants.REQUEST_ID

Reply:
This constant could be on this file, instead of Constants, to understand the context I guess. cc @mohak1283 @shiv07tiwari

Reply:
I think it will also work when this constant is declared in Constant.kt file. :+1: cc @shiv07tiwari

Reply:
@mohak1283 yes I'm sure it will work. I suggested this because in my opinion, is just to be aware of the context of the constant. But it can be on Constant.kt file as well, I'm sure that makes more sense if we start using that specific constant in more places.